modules/installation-vsphere-infrastructure.adoc

```diff
@@ -6,24 +6,29 @@
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 6.5
-or 6.7U2 or later instance.
+You must install the {product-title} cluster on a VMware vSphere version that meets the requirements for the components that you use.
 
-VMware recommends using vSphere Version 6.7 U2 or later with your {product-title}
-cluster.
-vSphere 6.7U2 includes:
+.Minimum supported vSphere version for VMware components
+|===
+|Component | Minimum supported versions |Description
 
-* Support for VMware NSX-T
-* Support for vSAN, VMFS and NFS, using the in-tree VCP
+|Hypervisor
+|vSphere 6.5 with HW version 13
+|This version is the minimum version that {op-system-first} supports. See the link:https://access.redhat.com/ecosystem/search/#/ecosystem/Red%20Hat%20Enterprise%20Linux?sort=sortTitle%20asc&vendors=VMware&category=Server[Red Hat Enterprise Linux 8 supported hypervisors list].
 
-While vSphere 6.5 with Hardware version 13 is supported, {product-title}
-clusters are subject to the following restrictions:
+|Networking (NSX-T)
+|n/a
+|vSphere 6.5 or vSphere 6.7U2+ are required for {product-title}. Because previous versions of vSphere with NSX-T are not currently compatible with {product-title}, NSX-T is not supported. NSX-T certification is in process and will be supported in a future release.
 
-* NSX-T SDN is not supported.
-* You must use another SDN or storage provider that {product-title} supports.
+|VMware CSI driver storage (vSphere CSI plug-in)
+|vSphere 6.7U3
+|This plug-in creates vSphere storage by using the standard Container Storage Interface. The vSphere CSI driver is supported by VMware.
 
-If you use a vSphere version 6.5 instance, consider upgrading to 6.7U2 before
-you install {product-title}.
+While previous vSphere versions do not provide the required API support for the CSI plug-in, Red Hat does continue to support them with in-tree storage drivers.
```

Comment:
@listey Can you review this wording, please? It's related to your comment here: https://github.com/openshift/openshift-docs/pull/19866#issuecomment-592036239

If it looks good, then we could also apply it to OCP 4.3+ (as outlined in that original PR).